_pages/fellowship_cohort_applications.md

```diff
@@ -6,6 +6,10 @@ title: Fellowship in Clinical AI - Cohort 3 Application (2024-25)
 Applications for Cohort 3 of the Fellowship in Clinical AI are now open.
 Successful applicants will be fellows in clinical AI from August 2024 - August 2025.
 
+<h3 style>Recruitment Regions</h3>
+![Recruitment region map](/assets/img/people/FCAI_Cohort3_map.png)
```

Comment:
i think this image looks too big on the page, i'd suggest replacing with :
`<div style="text-align: center"><img src="/assets/img/people/FCAI_Cohort3_map.png" width="70%"/></div><br><br>`